pkg/server/node_test.go

```diff
@@ -94,8 +94,6 @@ func createTestNode(
 	)
 	cfg.DB = client.NewDB(sender, cfg.Clock)
 	cfg.Transport = storage.NewDummyRaftTransport(st)
-	cfg.MetricsSampleInterval = metric.TestSampleInterval
```

Comment:
A bit of this was just driveby cleanup. Here, `TestStoreConfig` already set these. `server.Context` -> `server.Config` must have been missed before.